### PR TITLE
ci: run cron jobs on Sunday evening

### PR DESCRIPTION
**Pull request by smol-ninja**

### CODE REVIEW

**Comment by PaulRBerg** [comment]
Can we do it on Saturday at 1am?

So that there are 26 hours between this cron job and the one in V2 Core.

Many RPCs have 24-hour billing cycles.

**Comment by smol-ninja** [comment]
Done. Also changed time for `ci-fork` to make 1 AM reserved for periphery only.

File: .github/workflows/ci-fork.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: "CI Fork and Util tests"
 
 on:
     schedule:
-      - cron: "0 3 * * 1,3,5" # at 3:00 AM UTC on Monday, Wednesday and Friday
+      - cron: "0 1 * * 1,3,5" # at 1:00 AM UTC on Monday, Wednesday and Friday
```
Comment (PaulRBerg) [inline .github/workflows/ci-fork.yml:5]: Should we run this only once per week?

Since V2 Periphery is getting deprecated.

Comment (smol-ninja) [reply .github/workflows/ci-fork.yml:5]: Yeah sure but shouldn't we do that after the next release of Lockup? Until then, its the one with the most usage in production.

Comment (PaulRBerg) [reply .github/workflows/ci-fork.yml:5]: Good point

**Review by PaulRBerg — APPROVED** [review]
LGTM now, merging

**Pull request merged**